AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.html

```diff
@@ -172,7 +172,7 @@ <h2 class="govuk-heading-m">Time and location</h2>
               id="hearingStartTimeMinute"
               name="hearingStartTimeMinute"
               type="number"
-              min="1"
+              min="0"
```

Comment:
note: had to change this as when to set 1, you can't book hearings on the hour in angular 15 (15:00, 16:00 etc)